From 37431d6f12ba3c7f8a0f7f7eb21a67579544b1cc Mon Sep 17 00:00:00 2001
From: Nikolaj Bjorner <nbjorner@microsoft.com>
Date: Wed, 20 Nov 2019 15:01:04 -0800
Subject: [PATCH] Fix fail to build on clang++ and g++

Without this patch `src/util/mpz.cpp` fails to build when `__GNUC__` is
defined.
---
 src/util/mpz.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/util/mpz.cpp b/src/util/mpz.cpp
index a8190df1b..9c2d3d5ff 100644
--- a/src/util/mpz.cpp
+++ b/src/util/mpz.cpp
@@ -72,6 +72,8 @@ inline uint64_t _trailing_zeros64(uint64_t x) {
 
 #if defined(_WINDOWS) && !defined(_M_ARM) && !defined(_M_ARM64)
 // _trailing_zeros32 already defined using intrinsics
+#elif defined(__GNUC__)
+// _trailing_zeros32 already defined using intrinsics
 #else
 inline uint32_t _trailing_zeros32(uint32_t x) {
     uint32_t r = 0;
